### PR TITLE
Fix sympc tests

### PR DESCRIPTION
**Pull request by gmuraru**
## Description
Get a root client for each party that is part of the communication.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- The tests should run without any problem

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: src/sympc/module/nn/conv.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def share_state_dict(
         bias = None
         if ispointer(state_dict):
             weight = state_dict["weight"].resolve_pointer_type()
-            if "bias" in state_dict.keys().get():
+            if "bias" in weight.client.python.List(state_dict).get():
```
Comment (aanurraj) [inline src/sympc/module/nn/conv.py:91]: why we are doing this ?

Comment (gmuraru) [reply src/sympc/module/nn/conv.py:91]: We are looking or the "bias" term in the ```state_dict``` keys (if it is there, we know that we also have the bias)
`state_dict.keys().get()` returns `NotImplemented` (from the SyftCore side).
What we do here --> we create a remote list on the other side (client side) with the keys from the dictionary.

Ex:
```
list({1:3, 4:5}) --> [1,4]
```
and we `get` that list to check if it has `bias` as a key

**Review by aanurraj — APPROVED** [review]

**Review by LaRiffle — APPROVED** [review]
